Seminars/Sem.02/Pract.02/Task04.cpp

```diff
@@ -0,0 +1,123 @@
+#include <iostream>
+#include <fstream>
+
+const int RELATION_PAIRS_MAX_COUNT = 25;
+const int NUMBER_MAX_LENGTH = 11;
+const int BUFF_SIZE = 1024;
+const char* FILE_NAME = "relation.txt";
+
+struct Pair
+{
+	unsigned int a;
+	unsigned int b;
+};
+
+struct Relation
+{
+	unsigned int pairsCount;
+	Pair pairs[RELATION_PAIRS_MAX_COUNT];
+};
+
+void convertStringToInt(const char* str, int& num, const int numLength)
+{
+	int helpingNum = 1;
+	for (int i = numLength - 1; i >= 0; i--)
+	{
+		num += ((int)str[i]) * helpingNum;
+		helpingNum *= 10;
+	}
+}
+
+Pair* CreatePair(unsigned int a, unsigned int b)
+{
+	Pair pair{ a,b };
+	return &pair;
```

Comment:
BIG mistake. If this was at the exam, you would receive 0 points for the whole exam.